database/rrdvar.c

```diff
@@ -250,6 +250,23 @@ struct variable2json_helper {
     size_t counter;
 };
 
+static int single_custom_variable2json(void *entry, void *data) {
```

Comment:
Contains copy of code. Big no-no. Please refactor to at least something similar to the following:

```c
static int single_variable2json_worker(void *entry, void *data, int custom) {
    ...
}

static int single_custom_variable2json(void *entry, void *data) {
 return single_variable2json_worker(entry, data, 1);
}

static int single_variable2json(void *entry, void *data) {
    return single_variable2json_worker(entry, data, 0);
}
```

Reply:
@cakrit 
I used a different approach for refactoring. Please let me know if it is not ok.